step-templates/mysql-backup-database.json

```diff
@@ -0,0 +1,105 @@
+      "Octopus.Action.EnabledFeatures": ""
+    },
+    "Parameters": [
+      {
+        "Id": "7030ad54-4e10-4854-998a-ca91e3b490a3",
+        "Name": "createMySQLServerName",
+        "Label": "Server",
+        "HelpText": "Name or IP of the MySQL server",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "07dbf251-a982-4cc5-b6c1-dc0af50c519c",
+        "Name": "createusername",
```

Comment:
Can you namespace this parameter, e.g. `BackupMySQL.UserName` or something similar?